Gems/ExternalProfilers/SuperluminalProfiler/Code/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+set(SUPERLUMINAL_API_PATH "C:/Program Files/Superluminal/Performance/API" CACHE PATH "Path to the Superluminal 3rd party profiler API folder.")
+list(APPEND CMAKE_MODULE_PATH ${SUPERLUMINAL_API_PATH})
+
```

Comment:
as above it might be worth doing a message indicating that its a non free license, maybe even before it tries to find_package so that if there is an error finding package they go to the commercial site?